.changeset/six-students-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@iota/ledgerjs-hw-app-iota': minor
```

Comment:
this is a patch more than a minor update, and we could describe this change with a generic "Update dependencies"